src/showcase-merge/app/shared/meta.ts

```diff
@@ -0,0 +1,155 @@
+            { label: 'Sidebar', link: './components/sidebar', variantOnly: 'lean' },
+          ],
+        },
+        {
+          name: 'Layout Components',
+          entries: [
+            { label: 'Accordion', link: './components/accordion' },
+            { label: 'Breadcrumb', link: './components/breadcrumb' },
+            { label: 'Ghettobox', link: './components/ghettobox' },
+            { label: 'Notification', link: './components/notification' },
+            { label: 'Pagination', link: './components/pagination' },
+            { label: 'Processflow', link: './components/processflow' },
+            { label: 'Table', link: './components/table' },
+            { label: 'Tabs', link: './components/tabs' },
+            { label: 'Textexpand', link: './components/textexpand' },
+            { label: 'Usermenu', link: './components/usermenu' },
+          ],
+        },
+        {
+          name: 'Button & Indicator Components',
+          entries: [
+            { label: 'Badge', link: './components/badge' },
+            { label: 'Button', link: './components/button' },
+            { label: 'Icon', link: './components/icon' },
+            { label: 'Links', link: './components/links' },
+            { label: 'Loading', link: './components/loading' },
+          ],
+        },
+        {
+          name: 'Popup & Modal Components',
+          entries: [
+            { label: 'Dropdown', link: './components/dropdown' },
+            { label: 'Lightbox', link: './components/lightbox' },
+            { label: 'Tooltip', link: './components/tooltip' },
+          ],
+        },
+      ],
+    },
+    {
+      name: 'angular-maps',
+      packageName: '@sbb-esta/angular-maps',
+      svgIcon: 'kom:location-pin-map-small',
+      image: 'assets/maps.jpg',
+      description: 'Components to display 2D/3D maps and to provide map interaction..',
+      sections: [
+        {
+          name: 'Introduction',
+          entries: [
+            { label: 'Getting started', link: './introduction/getting-started' },
+            { label: 'Overview and usage', link: './introduction/overview-and-usage' },
+            { label: 'Mapping basics', link: './introduction/mapping-basics' },
+          ],
+        },
+        {
+          name: 'Maps',
+          entries: [
+            { label: 'WebMap', link: './components/esri-web-map' },
+            { label: 'WebScene', link: './components/esri-web-scene' },
+          ],
+        },
+        {
+          name: 'Map Utilities',
+          entries: [
+            { label: 'Layerlist', link: './components/esri-layer-list' },
+            { label: 'Basemap Gallery', link: './components/esri-basemap-gallery' },
+            { label: 'Legend', link: './components/esri-legend' },
+          ],
+        },
+      ],
+    },
+  ];
+
+  static libraries(): ShowcaseMetaLibrary[] {
+    return this.showcaseMetaLibraries;
+  }
+
+  static findByLibraryName(libraryName: string): ShowcaseMetaLibrary {
```

Comment:
Since we only have static data, should we maybe use maps instead of arrays in order to avoid these lookups?